### DISCUSSION TITLE
Can't connect my socket client with my socket server

### DISCUSSION THREAD

**Discussion by corzelito** [socketio/socket.io#4898] [Q&A] [answered]
Hello, I have an Angular app with Socket.io and a Node server also using Socket.io. Everything works perfectly on my local environment, but when I deploy it, the connection fails.

I'm attempting to deploy my server at '[www.midomain.com/proyect-syncro/socket.io](http://www.midomain.com/proyect-syncro/socket.io)'.

However, one of the requests is returning the following error:

{
  "message": "Invalid namespace"
}

Any guidance or assistance in resolving this issue would be greatly appreciated.

**Comment by darrachequesne** [comment, marked as answer]
For future readers: it seems the issue was due to a wrong path/namespace configuration.

Reference: https://socket.io/docs/v4/client-options/#path
